src/app/filters/filter-field.ts

```diff
@@ -12,6 +12,7 @@ import { FilterQuery } from './filter-query';
 export class FilterField {
   id?: string;
   placeholder?: string;
+  primaryQueries?: FilterQuery[];
```

Comment:
Considering the filter component is performing the filtering, why can we not make an internal copy of the queries property and filter that instead?

Having two lists in FilterField is confusing. The FilterField object is intended for developers to provide the filter config only. If you need a place to store a filtered list, we can store it internally. Or at least not include it as part of the FilterField type.